### DISCUSSION TITLE
release 2.4.2

### DISCUSSION THREAD

**Discussion by Aloren** [aerospike/spring-data-aerospike#538] [Team Posts]
Hi! Please publish a newly baked release to the Maven central.
Thanks.
@arrowplum it would be nice if somebody could discuss release process to maven central with us. we are ready to prepare automation on github actions for both projects.

**Comment by arrowplum** [comment]
Hi @Aloren,

We have an internal ticket for this at [OPS-4213] so we can refer to that to track the general effort here for enabling all community builds that need maven central access under `com.aerospike`. We will do another manual build if it is needed sooner.

I am sorry that this keeps being an issue.

Joe

**Comment by tdanylchuk** [comment]
thank you for the update, this release includes a critical fix, can we have a manual release for it this time and meanwhile prepare background\pipeline for the future releases?

**Comment by arrowplum** [comment]
https://oss.sonatype.org/service/local/repositories/releases/content/com/aerospike/spring-data-aerospike/2.4.2.RELEASE/spring-data-aerospike-2.4.2.RELEASE.pom

**Comment by tdanylchuk** [comment]
much appreciated.
